stable/aws-for-fluent-bit/templates/configmap.yaml

```diff
@@ -127,6 +127,29 @@ data:
         {{- end }}
 {{- end }}
 
+{{- if .Values.elasticsearch.enabled }}
+    [OUTPUT]
+        Name            es
+        Match           {{ .Values.elasticsearch.match }}
+        AWS_Region      {{ .Values.elasticsearch.region }}
```

Comment:
please make `.Values.elasticsearch.region` to be `.Values.elasticsearch.awsRegion` to match the configuration variable name

Reply:
ok